src/receipt/ReceiptPage.jsx

```diff
@@ -0,0 +1,463 @@
+              <div className="info-table">{this.renderOrderTable()}</div>
+              <div className="row">
+                <div className="order-total col-6 ml-auto">
+                  <ActionRow className="order-total-item border-divider">
+                    <span className="description">{this.props.intl.formatMessage(messages['ecommerce.receipt.table.order.subtotal'])}</span>
+                    <ActionRow.Spacer />
+                    <FormattedNumber
+                      value={this.props.order.total_before_discounts_incl_tax}
+                      style="currency" // eslint-disable-line react/style-prop-object
+                      currency={this.props.order.currency}
+                    />
+                  </ActionRow>
+                  {order.vouchers ? (
+                    <>
+                      <ActionRow className="order-total-item">
+                        <Badge variant="success">{this.props.intl.formatMessage(messages['ecommerce.receipt.table.order.discount'])}</Badge>
+                        <ActionRow.Spacer />
+                        {this.renderDiscountByType(order.basket_discounts)}
+                      </ActionRow>
+                      {order.enterprise_learner_portal_url ? (
+                        <div className="enterprise-customer">
+                          <FormattedMessage
+                            id="ecommerce.receipt.table.order.discount.message.enterprise.secondary"
+                            defaultMessage="Courtesy of {enterpriseName} is provided"
+                            values={{
+                              enterpriseName: order.basket_discounts[0].enterprise_customer_name,
+                            }}
+                          />
+                        </div>
+                      ) : null}
+                      <ActionRow className="border-divider" />
+                    </>
+                  )
+                    : null}
+                  <ActionRow className="order-total-item">
+                    <span>{this.props.intl.formatMessage(messages['ecommerce.receipt.table.order.total'])}</span>
+                    <ActionRow.Spacer />
+                    <FormattedNumber
+                      value={this.props.order.total_excl_tax}
+                      style="currency" // eslint-disable-line react/style-prop-object
+                      currency={this.props.order.currency}
+                    />
+                  </ActionRow>
+                </div>
+              </div>
+              {order.contains_credit_seat ? this.renderCreditMessaging() : null}
+              <ActionRow id="cta-nav-links">
+                <Hyperlink className="dashboard-link" destination={DASHBOARD_URL}>{this.props.intl.formatMessage(messages['ecommerce.receipt.link.dashboard'])}</Hyperlink>
+                <Hyperlink destination={FIND_COURSES_URL}>{this.props.intl.formatMessage(messages['ecommerce.receipt.link.find.courses'])}</Hyperlink>
+              </ActionRow>
```

Comment:
This action row actually has actions, so `ActionRow` could make sense here. Though, I'm assuming these might be better served as button links instead of `Hyperlink`, e.g.:

```jsx
<Button variant="link" href={DASHBOARD_URL}>Link text</Button>
```

Reply:
Remind me again why use a button in this case? `a11y` wise they are both focusable, wouldn't it be confusing to have a button that looks like a link for the user?

Reply:
`<Button variant="link">`  has a larger click target that's more in-line with a standalone CTA than  `Hyperlink` which is usually for a link inline with some other text (e.g., in a paragraph).

<img width="943" alt="image" src="https://user-images.githubusercontent.com/2828721/187760475-3ba82dc7-20c2-4cc6-8047-d0fbc96e141f.png">

Also, by using `Hyperlink` in `ActionRow`, the links are pretty much side-by-side without being able to distinguish at first glance that it's actually 2 separate links.